tests/contrib/unittest_plugin/test_unittest.py

```diff
@@ -0,0 +1,388 @@
+                self.assertTrue(2 == 2)
+                self.assertTrue('test string' == 'test string')
+                self.assertFalse('not equal to' == 'this')
+
+            @unittest.skip("another skip reason")
+            def test_will_be_skipped_with_a_reason(self):
+                self.assertTrue(2 == 2)
+                self.assertTrue('test string' == 'test string')
+                self.assertFalse('not equal to' == 'this')
+
+        suite = unittest.TestLoader().loadTestsFromTestCase(UnittestExampleTestCase)
+        unittest.TextTestRunner(verbosity=0).run(suite)
+
+        spans = self.pop_spans()
+        assert len(spans) == 4
+
+        assert spans[0].get_tag(test.TEST_TYPE) == SpanTypes.TEST
+        assert spans[0].get_tag(test.TEST_FRAMEWORK) == FRAMEWORK
+        assert spans[0].get_tag(SPAN_KIND) == KIND
+        assert spans[0].get_tag(COMPONENT) == COMPONENT_VALUE
+        assert spans[0].get_tag(test.TYPE) == SpanTypes.TEST
+        assert spans[0].get_tag(test.NAME) == 'test_will_be_skipped'
+        assert spans[0].get_tag(test.MODULE) == 'tests.contrib.unittest_plugin.test_unittest'
+        assert spans[0].get_tag(test.SUITE) == 'UnittestExampleTestCase'
+        assert spans[0].get_tag(test.TEST_STATUS) == test.Status.SKIP.value
+        assert spans[0].get_tag(test.SKIP_REASON) == ''
+
+        assert spans[1].get_tag(test.TEST_TYPE) == SpanTypes.TEST
+        assert spans[1].get_tag(test.TEST_FRAMEWORK) == FRAMEWORK
+        assert spans[1].get_tag(SPAN_KIND) == KIND
+        assert spans[1].get_tag(COMPONENT) == COMPONENT_VALUE
+        assert spans[1].get_tag(test.TYPE) == SpanTypes.TEST
+        assert spans[1].get_tag(test.NAME) == 'test_will_be_skipped_with_a_reason'
+        assert spans[1].get_tag(test.MODULE) == 'tests.contrib.unittest_plugin.test_unittest'
+        assert spans[1].get_tag(test.SUITE) == 'UnittestExampleTestCase'
+        assert spans[1].get_tag(test.TEST_STATUS) == test.Status.SKIP.value
+        assert spans[1].get_tag(test.SKIP_REASON) == 'another skip reason'
+
+        assert spans[2].get_tag(test.TEST_TYPE) == SpanTypes.TEST
+        assert spans[2].get_tag(test.TEST_FRAMEWORK) == FRAMEWORK
+        assert spans[2].get_tag(SPAN_KIND) == KIND
+        assert spans[2].get_tag(COMPONENT) == COMPONENT_VALUE
+        assert spans[2].get_tag(test.TYPE) == SpanTypes.TEST
+        assert spans[2].get_tag(test.NAME) == 'test_will_fail_first'
+        assert spans[2].get_tag(test.MODULE) == 'tests.contrib.unittest_plugin.test_unittest'
+        assert spans[2].get_tag(test.SUITE) == 'UnittestExampleTestCase'
+        assert spans[2].get_tag(test.TEST_STATUS) == test.Status.FAIL.value
+        assert spans[2].get_tag(test.SKIP_REASON) is None
+        assert spans[2].get_tag(ERROR_MSG) == 'False is not true'
+        assert spans[2].get_tag(ERROR_TYPE) == 'builtins.AssertionError'
+
+        assert spans[3].get_tag(test.TEST_TYPE) == SpanTypes.TEST
```

Comment:
These repeated assertions are another instance where a bit of deduplication would noticeably improve the changeability of this code.